lib/server.js

```diff
@@ -110,7 +110,9 @@ class S3Server {
                 log.info('initial health check succeeded', {
                     healthStatus: results,
                 });
-                this.startup();
+                // creates dual-stack endpoints for IPv4 and IPv6
+                this.startup('0.0.0.0', 8000);
+                this.startup('::1', 8001);
```

Comment:
Since we want to listen on all local IPV6 addresses, this should be `::` and not `::1`. Can you confirm if that works?